woocommerce/woocommerce
# add post tags support to docs manifest generator

### Changes proposed in this Pull Request:

This PR is based on https://github.com/woocommerce/woocommerce/pull/43468. It should be merged before this PR.

It adds support for front matter `tags` element on non `README.md` `docs/**/*.md` files.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check that the `tags` in the manifest match the tags added in #43468.
2. Run
```
pnpm --filter=monorepo-utils build
pnpm utils md-docs create docs woocommerce
diff manifest.json docs/docs-manifest.json
```
4. The `tags` lines should be the only differences (except possible hash changes).



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
